hl/tools/h5watch/h5watch.c

```diff
@@ -665,7 +665,7 @@ usage(const char *prog)
  *-------------------------------------------------------------------------
  */
 static void
-parse_command_line(int argc, const char *argv[])
+parse_command_line(int argc, char *argv[])
```

Comment:
Does this routine modify its arguments vector? If not, then it should be `const char * const * argv[]`.